README.md

```diff
@@ -1 +1,64 @@
-EXPERIMENTAL WIP Json library based on a port of the [Jawn Parser](https://github.com/non/jawn) built on Arrow.
+# Helios
+
+Json library based on a port of the [Jawn Parser](https://github.com/non/jawn) built on Arrow.
+
+## How to use it
+
+`Helios` uses kotlin `1.3.10` version and `Arrow` `0.8.1` version.
+
+To import the library on Gradle add the following repository and dependencies:
+
+```
+repositories {
+      maven { url "https://jitpack.io" }
+ }
+ 
+dependencies {
+    compile "com.47deg:helios-core:0.0.1-SNAPSHOT"
+    compile "com.47deg:helios-meta:0.0.1-SNAPSHOT"
+    compile "com.47deg:helios-parser:0.0.1-SNAPSHOT"
+}
+```
+
+Once it's imported, we just need to define our module on this way 
+
+```kotlin
+@json
+data class Person(val name: String, val age: Int) {
+  companion object
+}
+```
+
+The `@json` annotation will provide the decoder and encoder for that data class, so we are able to
+
+### Decode
+
+```kotlin
+val jsonStr = 
```

Comment:
We should use ank in these snippets

Reply:
How can I add Ank to be able to use it in the Readme?

Reply:
https://github.com/arrow-kt/arrow/tree/master/modules/ank